sda/local.go

```diff
@@ -398,6 +398,11 @@ func (l *LocalTest) NewLocalConode(port int) *Conode {
 
 }
 
+// NewContext creates a context from a ServerIdentity.
+func (l *LocalTest) NewContext(si *network.ServerIdentity, servID ServiceID, manager *serviceManager) *Context {
```

Comment:
Naming... It _is_ a constructor of a `Context`, as we're not returning an existing one, but rather we create a new context. The _real_ context that we would need is stored in the service, and it's not stored anywhere else, so we cannot retrieve it.